Sponsors.md

```diff
@@ -1,90 +1,86 @@
 # Sponsors
 
-We thought long and hard about accepting sponsorships. You will find our thinking in the “History” section, at the end of this page. 
+## Main way to sponsor us financially: through OpenCollective
 
-_These days, it seems, everyone has some advertisement targeted to software developers..._
+The main way to sponsor DevStaff is through [our OpenCollective page](https://opencollective.com/devstaff), which helps us gather financial contributions via a transparent system. There, you can find the available contribution options such as:
 
-Below you will find the available sponsorship packages / options: 
+* One-time contribution of a custom amount
+* One-time contributions of predefined amounts to sponsor a number of next meetups
 
-## DevStaff Swag Sponsor
-Help the community members wear the DevStaff community name with pride! With this option, you will sponsor the purchase of promotional material with the DevStaff brand, in return for awareness of your own brand.  (Please note we do not currently offer a co-branding option, see below for awareness benefits). All material will be made available to community and org team members.
+After contributing, you can be listed in OpenCollective's DevStaff contributors. 
+Also, as a thank you for sponsoring a number of the upcoming meetups, we will feature you in those meetups by mentioning you and your contribution.
+See the "Awareness benefits" section below for more details.
 
-The swag pack will be awarded to lucky attendees through a lottery that will take place at the end of the meetup. Your company will be publicly thanked for making this possible. 
+## Awareness benefits
```

Comment:
Thanks all! @nickorfas , @mzampetakis , I made a new section listing the standard awareness benefits here. Includes the new `"Thank you" mention in the announcement of the sponsored meetups.` you mentioned.

Note that the "Catering Sponsor" and "Venue Sponsor" sections mention two additional benefits (that I kept from before this PR): online banner & slide.